<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationWidgetBase</class>
 <widget class="QWidget" name="TranslationWidgetBase">
  <layout class="QVBoxLayout" name="layoutMain">
   <item>
    <layout class="QHBoxLayout" name="layoutInfo">
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <!--
       <property name="pixmap">
        <pixmap>../resources/icons/48-apps-kding.png</pixmap>
       </property>
       -->
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutInput">
       <item>
        <widget class="QLabel" name="labelSearch">
         <property name="text">
          <string>Enter the word or phrase you want to transl&amp;ate:</string>
         </property>
         <property name="buddy">
          <cstring>historyInput</cstring>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KHistoryComboBox" name="historyInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutButtons">
       <item>
        <widget class="KAnimatedButton" name="busyAnimation">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonTranslate">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KAnimatedButton</class>
   <extends>QToolButton</extends>
   <header>kanimatedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
